src/main/java/net/minecraftforge/client/gui/ModListScreen.java

```diff
@@ -341,8 +341,12 @@ public <T extends ObjectSelectionList.Entry<T>> void buildModList(Consumer<T> mo
 
     private void reloadMods()
     {
-        this.mods = this.unsortedMods.stream().
-                filter(mi->StringUtils.toLowerCase(stripControlCodes(mi.getDisplayName())).contains(StringUtils.toLowerCase(search.getValue()))).collect(Collectors.toList());
+        this.mods = this.unsortedMods
+                .stream()
+                .filter(mi ->
+                    StringUtils.toLowerCase(stripControlCodes(mi.getDisplayName()))
+                        .contains(StringUtils.toLowerCase(search.getValue()))
+                ).collect(Collectors.toList());
```

Comment:
This is kind of unnecessary given that it's just formatting but at this point who gives a shit. I only mention it because it's not present in the diff in #9860.